lms/djangoapps/grades/models.py

```diff
@@ -636,3 +637,55 @@ def get_override(cls, user_id, usage_key):
 def prefetch(user, course_key):
     PersistentSubsectionGradeOverride.prefetch(user.id, course_key)
     VisibleBlocks.bulk_read(user.id, course_key)
+
+
+class PersistentSubsectionGradeOverrideHistory(models.Model):
+    """
+    A django model tracking persistent grades override audit records.
+    """
+    PROCTORING = 'PROCTORING'
+    GRADEBOOK = 'GRADEBOOK'
+    OVERRIDE_FEATURES = (
+        (PROCTORING, 'Proctoring'),
+        (GRADEBOOK, 'Gradebook'),
+    )
+
+    CREATE_OR_UPDATE = 'CREATEORUPDATE'
+    DELETE = 'DELETE'
+    OVERRIDE_ACTIONS = (
+        (CREATE_OR_UPDATE, 'Create_or_update'),
+        (DELETE, 'Delete')
+    )
+
+    class Meta(object):
+        app_label = "grades"
+
+    override_id = models.IntegerField(db_index=True)
+    feature = models.CharField(
+        max_length=10,
```

Comment:
I would make this bigger (like 32), in case we want to add a choice that's a longer string some day.